spark-common/src/main/js/src_contentwin/spark-contentwindow.js

```diff
@@ -3,6 +3,91 @@ import 'iframe-resizer/js/iframeResizer.contentWindow';
 window.SPARK = window.SPARK || {};
 
 (function(sparkInstance, windowEl) {
+    var parentWindow = windowEl.parent;
+
+    /*
+     * Copy in the style elements from the parent pages head into this one.
+     * This only makes the design tokens available in this frame but does not enable theming (i.e. dark mode) yet.
+     */
+    function copyThemeTokens() {
+        // copy style tags into our app iframe if not already present
+        if (document.querySelector('style[data-theme]:not([data-custom-theme])')) {
+            return;
+        }
+
+        // look for data-theme style tags on parent document and copy those in
+
+        if (parentWindow && parentWindow.document) {
+            // copy theming CSS from outer window
+            parentWindow.document.querySelectorAll('style[data-theme]:not([data-custom-theme])')
+                .forEach((themeStyle) => {
+                    var theme = themeStyle.getAttribute('data-theme');
```

Comment:
Is there any reason for using `getAttribute('data-...')` here and `.dataset` in other places? Also applies to line 27.

Reply:
No reason, updated it now 👍